UndertaleModLib/Models/UndertaleGeneralInfo.cs

```diff
@@ -481,6 +481,14 @@ public void Unserialize(UndertaleReader reader)
         if (reader.ReadOnlyGEN8)
             return;
 
+        // TestForCommonGMSVersions is already run during the object counting phase, so the previous general info is always more accurate.
+        var prevGenInfo = reader.undertaleData.GeneralInfo;
+        Major = prevGenInfo.Major;
+        Minor = prevGenInfo.Minor;
+        Release = prevGenInfo.Release;
+        Build = prevGenInfo.Build;
+        Branch = prevGenInfo.Branch;
+        /*
```

Comment:
We probably want to just remove this old code, rather than commenting it out.

Reply:
agree